core/base/localizedTopologicalSimplification/LocalizedTopologicalSimplification.h

```diff
@@ -34,7 +34,11 @@
 
 namespace ttk {
 
-  std::string toFixed(const float &number, const int precision = 2);
+  std::string toFixed(const float &number, const int precision = 2) {
```

Comment:
Hi Jonas,
I moved this function's body into a .cpp file several PRs ago to avoid ODR violations (I had a warning with clang-tidy, check for "ODR" in the "Use clang-tidy" step of the "lint-code" CI job). 
I think that if you want to keep it in the header file, it might be better to declare it as inline (as also suggested by clang-tidy).

Reply:
You are right, I will at least inline/constexp it. I always wanted to move it to the debug class. Any thoughts on that?

Reply:
Why not, it may be useful outside of LTS. I always find cumbersome to use `setprecision` directly.